### DISCUSSION TITLE
Dev Protocol - YouTube channel

### DISCUSSION THREAD

**Discussion by vinzvinci** [dev-protocol/community#37] [Ideas]
# 🚀 We will start a YouTube channel for Dev Protocol 🌟

We will create a playlist per topic.

--- 

### 🔖 1st topic: All about Dev Protocol
- Episode 1: Introduction to Dev Protocol 
  - Intro
  - About Dev Protocol
  - What technology we have
  - Etc. 

- Episode 2 (Undecided)
  
### 🔖 2nd topic: All about stakes.social
- Episode 1: Getting started with stakes.social
  - Intro
  - About stakes.social
  - Benefits and etc.

- Episode 2
  - Intro
  - How to Staking
  - Etc.

- Episode 3
  - Intro
  - How to get ETH / DEV 
  - Etc.

### 🔖 3rd topic: Video Interviews
  - Intro
  - Interview developer `(who's using our technology)`
  - Etc.

We can update this, please suggest anything about Dev Protocol. What do you want to know? Feel free to suggest below 👍

**Comment by kazu80** [comment]
Since the number of developers entering Web3 as a result of the Dev Protocol will increase in near the future, it may be good to create content for those people.

- What is a blockchain?
- To have a token
- Tax

Such

**Reply by kazu80** [reply]
There are already a lot of videos on Youtube, so I felt that I could just put a link

**Reply by pdotall** [reply]
That was my solution writing this article!

https://community.devprotocol.xyz/t/blockchain-basics-for-creators/270/8

I'm updating it constantly, I will write down more things that were questioned today.

**Reply by kazu80** [reply]
Cool!!! Perfect!

**Reply by pdotall** [reply]
Do you have any suggestion for a topic?

**Comment by pdotall** [comment]
The more we can improve the information asymmetry the better. For the average Creator,  learning about blockchain is hard and can be too big of a challenge.

**Reply by vinzvinci** [reply]
Agree, blockchain is hard coz, not all school/university teaching that topic. So not all are aware of that technology, we need to create content that is easy for people to understand the topic. This is a big step ^^

**Reply by pdotall** [reply]
I think that the basics are easily available,  if we create an youtube channel it should be specific to Dev Protocol.

**Reply by vinzvinci** [reply]
Yeah, we should always relate it to Dev Protocol ^^
